### ISSUE TITLE
Accessibility - Top Navigation

### ISSUE THREAD

**Issue by sforschen** [hlxsites/elixirsolutions#78]
Add skip to main content link
 
Mark the active navigation item by using aria-current="page"
Reference: [Accessible Rich Internet Applications (WAI-ARIA) # aria-current](https://www.w3.org/TR/wai-aria-1.2/#aria-current)
 
Mobile navigation focus order (DOM order) should match visual order & flow of the page: left to right, top to bottom.
Issue: Search field should not be last in the menu tab order, it should come after the logo
Reference
•	[SC 2.4.3:Focus Order (Level A)](https://www.w3.org/WAI/WCAG21/Understanding/focus-order.html)
•	https://docs.deque.com/issue-help/1.0.0/en/tab-order-illogical

Search/Autocomplete Form Field

Input field should have a corresponding label
![image](https://github.com/hlxsites/elixirsolutions/assets/134433004/83103e3c-d3db-4292-b686-618279e7e672)
Needs better keyboard support – see recommended keyboard support in W3C
References
•	[W3C Combobox](https://www.w3.org/WAI/ARIA/apg/patterns/combobox/)
•	[GovUK Autocomplete example](https://alphagov.github.io/accessible-autocomplete/examples/)

**Comment by shsteimer** [comment]
I'll work on tab ordering, main issue is that elements and therefor tab ordering is different b/w desktop and mobile. but it's fixable

**Comment by shsteimer** [comment]
> Input field should have a corresponding label

Do you the label to be visible, or hidden for screen readers only? I assume hidden since sighted users have the placeholder as label?

**Comment by shsteimer** [comment]
Will for on addressing keyboard support, already started a bit of that in #83

**Comment by sforschen** [comment]
For search hidden should work, my accessibility person said "For Search fields, a visible label is usually not needed because the context of the field (i.e. search icon/button) makes its purpose clear. For most other input fields, a visible label is needed, as well as a programmed label. Placeholder text alone is typically not best practice because it goes away and can leave the user confused if they forget what the field is for."

**Comment by shsteimer** [comment]
PR above solves all of this except for: 
* aria-correct - lets discuss but I'm not sure we need it
* keyboard nav for search, which I'm working on now

**Comment by sforschen** [comment]
I think we are good with this I checked tabbing order both in mobile and desktop sizes and it works.
